### PR TITLE
[store pickup] code improvement

### PR DESCRIPTION
**Pull request by TBlindaruk**
### Description (*)

- see https://www.php.net/manual/en/migration71.new-features.php#migration71.new-features.mulit-catch-exception-handling
- improve readability of app/code/Magento/InventoryInStorePickupAdminUi/Ui/Component/Listing/Column/IsPickupLocationActive.php

### Fixed Issues (if relevant)

-

### Manual testing scenarios (*)

-

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by novikor — APPROVED** [review]

File: app/code/Magento/InventoryInStorePickupAdminUi/Controller/Adminhtml/Order/NotifyPickup.php [hunk 1]
```diff
@@ -110,11 +109,9 @@ private function initOrder(): OrderInterface
         $id = $this->getRequest()->getParam('order_id');
         try {
             $order = $this->orderRepository->get($id);
-        } catch (LocalizedException $e) {
-            if ($e instanceof NoSuchEntityException || $e instanceof InputException) {
-                $this->messageManager->addErrorMessage(__('This order no longer exists.'));
-                $this->_actionFlag->set('', self::FLAG_NO_DISPATCH, true);
-            }
+        } catch (NoSuchEntityException|InputException $e) {
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryInStorePickupAdminUi/Controller/Adminhtml/Order/NotifyPickup.php:112]: Where `InputException` is thrown? `OrderRepository` contract doesn't state there is one

File: app/code/Magento/InventoryInStorePickupAdminUi/Ui/Component/Listing/Column/IsPickupLocationActive.php [hunk 2]
```diff
@@ -21,14 +21,26 @@ class IsPickupLocationActive extends Column
...
+        }
+
+        return $this->normalizeData($dataSource);
+    }
+
+    /**
+     * @param array $dataSource
+     * @return array
+     */
+    private function normalizeData(array $dataSource):array
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryInStorePickupAdminUi/Ui/Component/Listing/Column/IsPickupLocationActive.php:39]: What is the reason for extracting this method?

File: app/code/Magento/InventoryInStorePickupAdminUi/Controller/Adminhtml/Order/NotifyPickup.php (hunk 1 above)
Comment (TBlindaruk) [reply app/code/Magento/InventoryInStorePickupAdminUi/Controller/Adminhtml/Order/NotifyPickup.php:112]: it is a question for @novikor )) I have just change code, without logic changes

File: app/code/Magento/InventoryInStorePickupAdminUi/Ui/Component/Listing/Column/IsPickupLocationActive.php (hunk 2 above)
Comment (TBlindaruk) [reply app/code/Magento/InventoryInStorePickupAdminUi/Ui/Component/Listing/Column/IsPickupLocationActive.php:39]: Hm,

So as for me 1 method did some checks of the preconditions, and return value... but they do not do any transformation by himself

Responsibility of this method - transforming

**Pull request merged**